Assets/Scripts/Game/UserInterfaceWindows/DaggerfallGuildServicePopupWindow.cs

```diff
@@ -0,0 +1,194 @@
+// Project:         Daggerfall Tools For Unity
+// Copyright:       Copyright (C) 2009-2017 Daggerfall Workshop
+// Web Site:        http://www.dfworkshop.net
+// License:         MIT License (http://www.opensource.org/licenses/mit-license.php)
+// Source Code:     https://github.com/Interkarma/daggerfall-unity
+// Original Author: Hazelnut
+// Contributors:    
+// 
+// Notes:
+//
+
+using UnityEngine;
+using DaggerfallWorkshop.Utility;
+using DaggerfallWorkshop.Game.UserInterface;
+using DaggerfallConnect;
+
+namespace DaggerfallWorkshop.Game.UserInterfaceWindows
+{
+    /// <summary>
+    /// Supported guild services.
+    /// </summary>
+    public enum GuildServices
+    {
+        // Mages Guild:
+        MG_Buy_Spells = 60,
+        MG_Training = 61,
+        MG_Teleportation = 62,
+        MG_Quests = 63,
+        MG_Make_Spells = 64,
+        MG_Buy_Magic_Items = 65,
+        MG_Daedra_Summoning = 66,
+        MG_Identify = 801,
+        MG_Make_Magic_Items = 802,
+
+        // Fighters Guild:
+        FG_Training = 849,
+        FG_Repairs = 850,
+        FG_Quests = 851,
+
+        // Temples:
+        Make_Donation = 810,
+        Cure_Diseases = 813,
+    }
+
```

Comment:
I'm thinking of adding enums for all the faction IDs, to be accessible from anywhere, as things are called by faction ID for all kinds of things. Is it OK with you do to that and remove this section?

Reply:
Never mind, I changed my mind, and I don't think I would have needed to delete this section anyway.

Reply:
I missed this, sorry. Glad you change your mind, because I looked as how Gav was thinking of doing it, and realised how big the case statement would be if the guild and the service are separated for this. This way, the enum uses faction id tell the popup which service it is for, and the guild is a separate piece of data.